packages/react-core/src/components/Button/examples/Button.md

```diff
@@ -14,70 +14,109 @@ import ArrowRightIcon from '@patternfly/react-icons/dist/esm/icons/arrow-right-i
 ```
 
-### Disabled
+| Variant | Description|
+| --- | ---|
+| Primary | Primary buttons are the most visually prominent variant. Use for the most important call to action.|
+| Secondary | Secondary buttons are less visually prominant than primary buttons. Use for general actions on a page that require less emphasis than primary buttons. |
+| Tertiary | Tertiary buttons are the least visually prominent variant. Use to retain a classic button format with less emphasis than primary or or secondary buttons.  |
+| Danger | Danger buttons may be used for actions that are potentially destructive or difficult/impossible to undo. Danger an independent variant, but all button variants may use the the `isDanger` prop to apply similar styling. |
+| Warning | Warning buttons may be used for actions that change an important setting or behavior, but not in a destructive or irreversible way. |
+| Link | Links are labeled, but have no background or border. Use for actions that require less emphasis, actions that navigate users to another page within the same window, and/or actions that navigate to external pages in a new window. Links may be placed inline with text using the `isInline` prop.|
+| Plain | Plain buttons have no styling and are intended to be labeled with icons. |
+| Control | Control buttons can be labeled with text or icons. Primarily intended to be paired with other controls in an [input group](https://www.patternfly.org/v4/components/input-group). | 
+
+### Disabled buttons
+
+To indicate that an action is currently unavailable, all button variations can be disabled using the `isDisabled` prop.
 
 ```ts file="./ButtonDisabled.tsx"
 ```
 
-### Aria disabled
+### Progress indicators
 
-```ts file="./ButtonAriaDisabled.tsx"
+Progress indicators can be added to buttons to identify that an action is in progress after a click. 
+
+```ts file="./ButtonProgress.tsx"
 ```
 
-### Aria disabled button with tooltip
+### Small buttons
 
-```ts file="./ButtonAriaDisabledTooltip.tsx"
+To fit into tight spaces, primary, secondary. tertiary, danger, and warning button variations can be made smaller using the `isSmall` prop.
+
+```ts file="./ButtonSmall.tsx"
 ```
 
-### Aria disabled link as button with tooltip
+### Call to action (CTA) buttons
 
-```ts file="./ButtonAriaDisabledLinkTooltip.tsx"
+CTA buttons and links direct users to complete an action. Primary, secondary, tertiary, and link button variants can be styled as CTAs using the `isLarge` prop. 
+
+```ts file="./ButtonCallToAction.tsx"
+```
+
+### Block level buttons
+
+Block level buttons span the full width of the parent element and can be enabled using the `isBlock` prop.
+
+```ts file="./ButtonBlock.tsx"
+```
+
+### React button types
+
+React button types can be changed from default as needed. 
+
+```ts file="./ButtonTypes.tsx"
 ```
 
+## Links 
+
 ### Links as buttons
 
+Buttons that link to another resource may take the form of primary, secondary, tertiary, or link variants.
```

Comment:
Maybe something along the lines of:

```suggestion
Buttons that link to another resource may take the form of primary, secondary, tertiary, or link variants. This can be achieved by passing in `component="a"` and an `href` prop.
```

To describe how to use links as buttons.